.gitmodules

```diff
@@ -13,3 +13,6 @@
 [submodule "extern/network_backend/ns3"]
 	path = extern/network_backend/ns3
 	url = https://github.com/astra-sim/astra-network-ns3.git
+[submodule "inputs/workload/execution-graph-converter-Intel"]
+	path = inputs/workload/execution-graph-converter-Intel
+	url = git@github.com:astra-sim/execution-graph-converter-Intel.git
```

Comment:
As the name suggests, we are not allowed to release this repository. We don't have any plan to make it public. Which tool do you need for this PR?